Shared/Samples/Create and save KML file/CreateAndSaveKMLView.swift

```diff
@@ -70,7 +70,8 @@ struct CreateAndSaveKMLView: View {
 
 extension CreateAndSaveKMLView {
     /// A KMZ file that can be used with the native file exporter.
-    final class KMZFile: FileDocument {
+    @MainActor
+    final class KMZFile: @preconcurrency FileDocument, Sendable {
```

Comment:
Not sure if this is the right fix for `FileDocument` conforming type, but it works.

Reply:
No, this is not the right approach because `FileDocument` has adopted concurrency (as evidenced by the Sendable conformance and preconcurrency attribute). Instead, work through the issues. Some can be addressed by adding the `nonisolated` keyword, but others may be more involved. Same with `CreateMobileGeodatabaseView.GeodatabaseFile`. Let me know if you'd like some help!